variables.tf

```diff
@@ -178,4 +195,16 @@ variable "vpn_connection_tunnel2_startup_action" {
   type        = string
   description = "The action to take when the establishing the tunnel for the second VPN connection. By default, your customer gateway device must initiate the IKE negotiation and bring up the tunnel. Specify start for AWS to initiate the IKE negotiation. Valid values are add | start."
   default     = "add"
-}
\ No newline at end of file
+}
+
+variable "vpn_tunnel_logging" {
+  type = bool
+  description = "Enable logging for the VPN tunnel"
+  default = true
+}
+
+variable "vpn_tunnel_logging_retention" {
```

Comment:
Please be sure to run a `terraform fmt` locally before committing your changes. Thanks!